x/evm/keeper/querier.go

```diff
@@ -500,6 +481,21 @@ func queryDepositState(ctx sdk.Context, k types.ChainKeeper, n types.Nexus, para
 	}
 }
 
+// QueryPendingCommands returns the pending commands for a gateway
+func QueryPendingCommands(ctx sdk.Context, keeper types.ChainKeeper, n types.Nexus) (types.PendingCommandsResponse, string, codes.Code) {
+	var resp types.PendingCommandsResponse
+	commands := keeper.GetPendingCommands(ctx)
```

Comment:
Why not merge this method into the grpc_query method? We're migrating all methods from querier.go to grpc_query.go I believe.

Reply:
moved into `grpc_query.go`